src/components/SarcoModal.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Button,
+    Text,
+    useDisclosure,
+    Box,
+    Flex,
+} from '@chakra-ui/react';
+import { colors } from 'theme/colors';
+
+interface ModalButtonProps {
+    label: string;
+    onClick: () => void;
+    dismissesModal?: boolean;
+}
+
+interface SarcoModalProps {
+    isDismissible?: boolean;
+    coverImage?: JSX.Element;
+    title: string;
+    subtitle?: string;
+    primaryButton: ModalButtonProps,
+    secondaryButton?: ModalButtonProps,
+}
+
+function useSarcoModal() {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+
+    const modal = (props: SarcoModalProps) =>
+        <Modal closeOnOverlayClick={props.isDismissible} isOpen={isOpen} onClose={onClose} isCentered>
+            <ModalOverlay />
+            <ModalContent minWidth='484px' paddingX='46px' paddingY='26px' bgColor={colors.brand[100]}>
```

Comment:
Would love some scrutiny on my use of styling in this PR. Didn't feel very consistent, as I mostly just poked and prodded till the thing looked right onscreen.

Reply:
I think this is just fine. We may want to revisit when we go through the app to make it more responsive for mobile.